packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -121,7 +121,8 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
     onGoToStep: null as WizardStepFunctionType,
     width: null as string,
     height: null as string,
-    footer: null as React.ReactNode
+    footer: null as React.ReactNode,
+    onClose: () => undefined as () => void
```

Comment:
Nitpick: `() => undefined as any` or `(): any => undefined` to be consistent with the rest of our code.